axelor-account/src/main/java/com/axelor/apps/account/service/move/massentry/MassEntryMoveCreateServiceImpl.java

```diff
@@ -123,6 +127,14 @@ public Move generateMassEntryMove(Move move) throws AxelorException {
 
         moveLineComputeAnalyticService.generateAnalyticMoveLines(moveLine);
         moveLineMassEntryRecordService.setAnalytics(moveLine, moveLineElement);
+        if (ObjectUtils.notEmpty(moveLineElement.getAnalyticMoveLineList())) {
+          moveLineElement
+              .getAnalyticMoveLineList()
+              .forEach(
+                  analyticMoveLine ->
+                      moveLine.addAnalyticMoveLineListItem(
+                          analyticMoveLineRepository.copy(analyticMoveLine, false)));
```

Comment:
Can you rename `moveLine` and `moveLineElement` so that it's more understandable ? Like `newMoveLine` or `moveLineCopy` or something

Reply:
Resolved